DateModule.js

```diff
@@ -0,0 +1,190 @@
+//coder: Katja Borchert
+
+//A Date Utility Module in the "Revealing Pattern Module" format
+/* eslint-disable no-unused-vars */
+var myObj = (function iife() {
+/* eslint-enable no-unused-vars */
+
+  //private variables:
+  var date = null;
+  
+  //private function: date instantiation
+  function _initDate(){
+    date = new Date();
+  }
+    
+  //public API:
+  
+  //setDate(): sets the internal date based on user input
+  //takes in: empty argument list, date object, or number)
+  function setDate(enteredDate) {
+  
+    //check that date is set: if not, initialize date to now
+    if (date === null) {
+      _initDate();
+    }
+	
+	/*checks the type of the parameter enteredDate:
+	check if the enteredDate is of type number
+	(excluding variables of value 'NaN')*/
+    if (typeof enteredDate === 'number' && !isNaN(enteredDate)){
+      date.setTime(enteredDate);
+    } else if (enteredDate instanceof Date){
+      date.setTime(enteredDate.getTime());
+    } else if (arguments.length === 0){
+      date.setTime(Date.now());
+	//in any other case alert user
+    } else {
+      alert('argument was of the wrong type');
+    }
+  }
+  
+  /*getDate(): returns the internally stored date; default is milliseconds
+  takes in: empty argument list, object with
+  property format set to "milliseconds" or "formatted")*/
+  function getDate(objFormat){
+  
+    //check that date is set: if not, initialize date to now
+    if (date === null){
+      _initDate();
+    }
+	
+	/*check if argument list is empty;
+	if so declare objFormat as an object with property format = "milliseconds"*/
+    if (arguments.length === 0){
+      objFormat = {};
+      objFormat.format= 'milliseconds';
+    }
+	
+	/*ensure that arguments are of the correct format: milliseconds or formatted
+	(if 'format' is undefined: make an alert)*/
+    if (objFormat.format === undefined){
+      alert('undefined format handed in');
+    } else if (objFormat.format ==='milliseconds'){
+      return date.getTime();
+    } else if (objFormat.format ==='formatted'){
```

Comment:
The output from this method should take the following format when `{ format: 'formatted' }` is passed to it:

[Month] [Date], [Year] - e.g. April 25, 2016

This also means your implementation of `isToday` is based on an incorrect implementation of this method.